docs/ml-clustering.md

```diff
@@ -11,6 +11,77 @@ In this section, we introduce the pipeline API for [clustering in mllib](mllib-c
 * This will become a table of contents (this text will be scraped).
 {:toc}
 
+## K-means
+
+K-means clustering with support for multiple parallel runs and a k-means++ like initialization mode 
```

Comment:
There is no longer support for multiple parallel runs, at least for now.

Also, could you include the same links as in the text in mllib-clustering.md?